src/dfx/src/commands/start.rs

```diff
@@ -525,3 +535,16 @@ pub fn empty_writable_path(path: PathBuf) -> DfxResult<PathBuf> {
         .with_context(|| format!("Unable to write to {}", path.to_string_lossy()))?;
     Ok(path)
 }
+
+fn check_docker_command_available() -> DfxResult<()> {
+    if std::process::Command::new("docker")
+        .args(["--version"])
```

Comment:
Even if the command is available, docker might still fail if the daemon is not running.